<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSS::BackgroundOptions</class>
 <widget class="QDialog" name="DSS::BackgroundOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Background Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Calibration Method</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbLinear">
          <property name="text">
           <string>Linear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbRational">
          <property name="text">
           <string>Rational</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="laCalibration">
        <property name="minimumSize">
         <size>
          <width>204</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>RGB Background Calibration Method</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="rbNone">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbMinimum">
          <property name="text">
           <string>Minimum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbMiddle">
          <property name="text">
           <string>Middle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbMaximum">
          <property name="text">
           <string>Maximum</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="laRGBCalibration">
        <property name="minimumSize">
         <size>
          <width>204</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
